src/app/apps/rup/mapa-camas/views/censos/censo-mensual/censo-mensual.component.ts

```diff
@@ -191,6 +191,9 @@ export class CensosMensualesComponent implements OnInit {
 
         const totalEgreso = this.totales.egresosAlta + this.totales.egresosDefuncion;
         this.datosCensoTotal['promDiasEstada'] = (totalEgreso === 0) ? '0' : (this.totales.diasEstada / totalEgreso).toFixed(2);
+
+        const porcentajeCamaOcupadas = (this.censo.length > 0) ? ((this.totales.pacientesDia / this.totales.disponibles24) * 100).toFixed(2) : '0';
+        this.datosCensoTotal['promCamasOcupadas'] = porcentajeCamaOcupadas;
```

Comment:
Es porcentaje, por ahí la variable promCamasOcupadas confunde porque no se saca un promedio.
Yo pondría algo así:

```suggestion
        this.datosCensoTotal['porcentajeCamaOcupadas'] = this.censo.length ? ((this.totales.pacientesDia / this.totales.disponibles24) * 100).toFixed(2) : '0';
```

Reply:
Corregido!

Reply:
Me da 0 siempre
![imagen](https://user-images.githubusercontent.com/56874534/170690437-2fe3bf14-d81d-4cd1-8b46-7b1c44c1a3c5.png)

Reply:
Corregido, ahora se muestran